tests/product/test_server_install.py

```diff
@@ -191,12 +191,12 @@ def test_install_with_java8_home(self, dummy=True):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_failure_without_java8_home(self, dummy=True):
+    def test_install_failure_without_java8_home(self):
         for container in self.cluster.all_hosts():
             self.cluster.exec_cmd_on_host(container,
                                           "mv /usr/java/jdk1.8.0_40 /usr/")
         self.upload_topology()
-        cmd_output = self.installer.install(dummy)
+        cmd_output = self.installer.install(dummy=True, pa_raise_error=False)
```

Comment:
Fixed package install and slider slider_install. Re-ran tests.product.test_package_install and tests.product.yarn_slider.